### ISSUE TITLE
figure out how to add pdf support

### ISSUE THREAD

**Issue by glennhartmann** [GoogleChromeLabs/llaminator#5]
@EiraGe

**Comment by tannerdolby** [comment]
👋  @glennhartmann Would the pdf support be mainly for .pdf files that are images (posters etc) that would be expected to show the image in browser via `imgElement.src`? Or prepare for any type of pdf (not specifically images) and display a more "generic" pdf related template image in browser along with the filename and/or link to the pdf blob url? 

The reason I ask is because I spun up a demo locally by adding `.pdf` to the accepted types for the input e.g. `<input ... accept="image/*, .pdf">` and then prototyping within `index.js`, I found if we pass the blob to `window.URL.getObjectUrl(b)` and assign it to the `imgElement` src for uploaded .pdf files, it doesn't render an image in the browser like the png/jpeg images would because the image `src` has a blob with MIME type `application/pdf`. When the MIME type is `image/*` the `<img src="blob">` renders in browser as expected. 

If we use something to convert the pdf blob to another image format, then it could be passed straight into the `<img>` elements `src`, otherwise I was having trouble displaying any sort of "preview" for uploaded .pdf files as the blob contains the uploaded .pdf and using `window.open(b)` confirms that as well as navigating to the blob URL, but using it in the img element src attribute doesn't render as the other image formats do.

**Comment by glennhartmann** [comment]
Yeah the idea would be PDFs that are basically just images (or static, non-interactive content at least). Not sure how we would want to handle multi-page documents yet (just use the first page?). We may be able to make it work on desktop by opening the pdf in its own iframe instead of using the existing imgElement, but I don't think that works on mobile.
